src/test/java/com/github/lombrozo/testnames/rules/RuleTestCaseContainsMockeryTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022-2024 Volodya Lombrozo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.github.lombrozo.testnames.rules;
+
+import com.github.lombrozo.testnames.Complaint;
+import com.github.lombrozo.testnames.TestCase;
+import com.github.lombrozo.testnames.javaparser.JavaTestClasses;
+import org.cactoos.list.ListOf;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link RuleTestCaseContainsMockery}.
+ *
+ * @since 1.3.4
+ */
+final class RuleTestCaseContainsMockeryTest {
+
+    @Test
+    void complaintsAboutMockery() {
+        final TestCase test = new ListOf<>(
```

Comment:
@h1alexbel I would highly recommend to avoid using redundant variables in the code (like the `test` variable). It's better to inline all the redundant variables. The same problem arises in `Mockery.java`, `MockeryTest.java`, `JavaParserTestCaseTest.java` classes.

You can read about redundant variables and why it's bad to have them right [here](https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html).